client/src/components/Winner/GameWinner.tsx

```diff
@@ -1,33 +1,39 @@
+import "./GameWinner.scss";
 import { Player } from "../../types/player/player.types";
 
 interface GameWinnerProps {
   players: Player[];
 }
 
 const GameWinner: React.FC<GameWinnerProps> = (props) => {
-  const players = props.players;
+  const { players } = props;
 
-  if (players.length === 1) {
-    return (
-      <>
-        {players[0] && (
-          <div className="game-winner">
-            Congratulations {players[0].name}! You caught them all!
-          </div>
-        )}
-      </>
-    );
-  } else {
-    const playersString = players.map((p) => p.name).join(", ");
+  const singleWinner = players.length === 1;
+  const winnerNameString = singleWinner
+    ? players[0].name
+    : players.map((p) => p.name).join(", ");
```

Comment:
Does this all need to change? The previous way was readable too?

Reply:
![Screenshot 2024-01-10 131253](https://github.com/TeamTopTrumps/top-trumps-game/assets/5334385/f25efbfc-232c-4b3d-bb1c-72b2cad6a79a)
![Screenshot 2024-01-10 131446](https://github.com/TeamTopTrumps/top-trumps-game/assets/5334385/220baeb0-57ee-4265-b651-0ea0649acc92)
I don't think they're too different? But I can see how the if else is maybe more readable than the ternary? So I can switch it to that if preferred?

Reply:
I'm fine with either - I'm less of a fan of the `const singleWinner = players.length === 1;` and this bit below `return (
    <>
      {players[0] &&
        (singleWinner ? (`  feels complex. 
        
        but both look solved with the above?